app/static/tests/unit/csvUpload.test.ts

```diff
@@ -3,6 +3,7 @@ import { csvUpload } from "../../src/app/csvUpload";
 
 describe("CSVUpload", () => {
     const file = { name: "testFile" } as any;
+    const fileTimeout = 20;
```

Comment:
We do need to give the mock file reader a small timeout or else tests sometimes haven't committed when we check them.